apps/onlyoffice.sh

```diff
@@ -23,6 +23,51 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+## Onlyoffice for NC 18.0.1 onwwards
+# Check for the NC-version
+CURRENTNCVERSION="$(occ_command config:system:get version | sed 's/^\(.*\..*\..*\)\..*/\1/')"
+CURRENTNCVERSION="${CURRENTNCVERSION//./}"
+if [ "$CURRENTNCVERSION" -lt "9999" ]
+then
+    CURRENTNCVERSION="$(sed 's/./&0/3' <<<"$CURRENTNCVERSION")"
```

Comment:
if you have a norml version like 1801, you then add here a zero to the 3rd character of 1801 so that you get 18001 back.
Because if later on you would have e.g. 18.0.10, the version would be 18010 which would be greater then the number that is checked later on. So I add a needed zero here.

Reply:
The only thing we care about here really is 18. Not the 18.X.X.X

Reply:
no, that is not right. we need 18.0.1 since on 18.0.0 there was a timeout issue when installing this.

Reply:
OK, you're right, but we already have a function for checking versions: https://github.com/nextcloud/vm/blob/master/lib.sh#L858-L877

Reply:
Okay, but I don't get how this works. Can you show me an example? Also is the current sub-version provided as an integer? Or how does the comparison of sub-versions work with that function?

Reply:
Actually I don't remember completely either, but it's used when comparing current versions. So I think something like this should work.

I'm testing now on my dev. 

```
if version_gt $CURRENTVERSION 18.0.1
then
    echo "continue"
fi
```

Btw, I would like to release 18.0.1 tomrorow, fyi.

Reply:
Result:

```
 CURRENTNCVERSION=18001
+ version_gt 18.0.1 18.0.1
+ local v1 v2 IFS=.
+ read -ra v1
+ read -ra v2
+ printf -v v1 %03d 18 0 1
+ printf -v v2 %03d 18 0 1
+ [[ 018000001 > 018000001 ]]
+ exit
```